setup.py

```diff
@@ -356,6 +363,9 @@
     'obspy.plugin.inventory.CSS': [
         'writeFormat = obspy.io.css.station:_write_css',
         ],
+    'obspy.plugin.inventory.SEISAN': [
+        'writeFormat = obspy.io.nordic.core:inventory_to_seisan',
```

Comment:
Hmm.. we also already have an `obspy.io.seisan` submodule (for waveforms), so maybe it's worth a thought to move the SEISAN output part there?